docs/reference/commandline/history.md

```diff
@@ -54,3 +55,42 @@ IMAGE               CREATED             CREATED BY
 c69cab00d6ef        5 months ago        /bin/sh -c #(nop) MAINTAINER Lokesh Mandvekar   0 B
 511136ea3c5a        19 months ago                                                       0 B                 Imported from -
 ```
+
+### Format the output
+
+The formatting option (`--format`) will pretty print history output
+using a Go template.
+
+Valid placeholders for the Go template are listed below:
+
+| Placeholder | Description|
+| ---- | ---- |
+| `.ID` | Image ID |
+| `.CreatedSince` | Elapsed time since the image was created if --human=true, otherwise timestamp of when image was created |
+| `.CreatedAt` | Timestamp of when image was created |
+| `.CreatedBy` | Command that was used to create the image |
+| `.Size` | Image disk size |
+| `.Comment` | Comment for image |
```

Comment:
Can you update the doc to align the table like below so that it is easy to update in the future?
```
| Placeholder     | Description |
| --------------- | ----------- |
| `.ID`           | Image ID    |
| `.CreatedSince` | Elapsed time since the image was created if --human=true, otherwise timestamp of when image was created |
| `.CreatedAt`    | Timestamp of when image was created |
| `.CreatedBy`    | Command that was used to create the image |
| `.Size`         | Image disk size |
| `.Comment`      | Comment for image |

```